mtp_api/apps/security/views.py

```diff
@@ -497,6 +497,9 @@ class CheckListFilter(django_filters.FilterSet):
     credit_resolution = django_filters.CharFilter(
         field_name='credit__resolution', lookup_expr='exact',
     )
+    actioned_by = django_filters.BooleanFilter(
+        field_name='actioned_by', lookup_expr='isnull', exclude=True,
```

Comment:
how come you need the `exclude=True`? would you not just post {actioned_by=false} to filter for anything that has a non-null actioned_by field?

Reply:
also, could be worth having a test that uses this filter

Reply:
I feel that this exposes the complexity to the the noms_ops app as it makes it confusing to read `params['actioned_by'] = False` rather than `params['actioned_by'] = True`. By doing `params['actioned_by'] = True` makes it very explicit that you are filtering for checks with ab `actioned_by `. This way just hides the complexity in the API rather than the front end.

Reply:
Yeah, it was a toss up between clean code and an easier to understand API - filtering by not null isn't directly possible, so we thought hiding the complexity in the API was better than having `actioned_by = False` return rows where actioned_by is a truthy value. 

https://django-filter.readthedocs.io/en/master/guide/tips.html#filtering-by-null-values

Unless you have a different solution, happy to try it if you think there's a less complex way that makes sense in the API calls too.

Reply:
filtering by "not null" would be possible if you add it to the fields bit (similar to [here](https://github.com/ministryofjustice/money-to-prisoners-api/blob/master/mtp_api/apps/credit/views.py#L221)) but your way of doing it is fine. i agree that it's a toss up between the client side being clear (actioned_by=True means that somebody actioned) or the api being clear ("exclude nulls if filter is set to true").